apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/components/MatrixQuestionForm.tsx

```diff
@@ -211,6 +236,31 @@ export const MatrixQuestionForm = ({
               <span>Add column</span>
             </Button>
           </div>
+          <div className="flex flex-1 items-center justify-end gap-2">
+            <Select
```

Comment:
Since we have the same functionality for multi select and ranking, can you create a reusable component for this in packages/ui/ShuffleOptionSelect 😊

Reply:
Sure